src/api-reference/accounting-integration/v4.accountingintegration-schema.markdown

```diff
@@ -180,8 +192,9 @@ Name|Type|Format|Description
 Name|Type|Format|Description
 ---|---|---|---
 `erpId`|`string`|`UUID`|Universally unique identifier of the ERP.
+`ledger`|-|`Ledger`| (**Currently not supported, future enhancement**)Concur Ledger Information.
```

Comment:
Hi @YezhenXue. Please remove all mention of future enhancements. If this is not currently available, we cannot advertise it as functionality, even in the future.

Reply:
done, please review it again